sdk/src/clearingHouseUser.ts

```diff
@@ -289,51 +312,55 @@ export class ClearingHouseUser {
 		return freeCollateral.gte(ZERO) ? freeCollateral : ZERO;
 	}
 
-	public getInitialMarginRequirement(): BN {
-		const postionMarginRequirement = this.getUserAccount().positions.reduce(
-			(marginRequirement, marketPosition) => {
+	/**
+	 * @returns The margin requirement of a certain type (Initial or Maintenance) in USDC. : QUOTE_PRECISION
+	 */
+	public getMarginRequirement(type: MarginCategory): BN {
+		return this.getUserAccount()
+			.positions.reduce((marginRequirement, _position) => {
```

Comment:
using this `_` syntax at beginning of variable name typically means it is unused (in ts and in rust) so i'd prefer not to call this `_position` since it is used